Examples/Mirrorator/README.md

```diff
@@ -0,0 +1,61 @@
+# iOS Mirrorator 
+
+Send data over Bluetooth to your Mirror templates!
+
+## Build and run 🏃‍
+
+You'll require the [react native cli](https://facebook.github.io/react-native/docs/getting-started.html) to build and run the project.
+Run:
+> `$ brew install node`
+
+> `$ brew install watchman`
+
+> `$ npm install -g react-native-cli`
+
+Lastly, from root of the project run:
+> `$ npm install`
+
+Now you can build and run the example project from Xcode on your iDevice!
+
+## Example
+
+Fill the key-value pairs with whatever string data you wish to send to your Mirror.
+
+<img src="Preview.PNG" alt="Preview" style="width: 300px;">
+
+Then, on tapping send, your hypothetical `clock` template that is located on Mirror prints the message in console in the listener closure:
```

Comment:
Maybe instead of operating on the hypothetical, we should give them an actual example to work with? e.g., "give it a try, and invoke the Flight Info example you might know from the app, set the template to X and data to X"

Reply:
My thought exactly, I just wasn't sure if we want to disclose that there is the `estimote/{demo}` template option.